sql_execution_engine/manual_engine.go

```diff
@@ -53,25 +103,33 @@ func (engine *manualSQLEngine) Execute(msgBatch []*core.Msg, tableDef *schema_st
 	}
 
 	query := stringBuilder.String()
-	_, err := engine.db.Exec(query, sqlArgs...)
+
+	if engine.cfg.SQLAnnotation != "" {
+		query = fmt.Sprintf("%s%s", engine.cfg.SQLAnnotation, query)
```

Comment:
this usage is not compatible with the current (proxy)annotation usage, which just configure string inside /**/.

e.g.
```go
if probeSQLAnnotation != "" {
		probeSQLAnnotation = fmt.Sprintf("/*%s*/", probeSQLAnnotation)
}